baseapp/src/main/java/io/trino/gateway/baseapp/BaseApp.java

```diff
@@ -6,22 +6,20 @@
 import com.google.inject.Guice;
 import com.google.inject.Injector;
 import com.google.inject.Module;
-
-import io.dropwizard.Application;
-import io.dropwizard.Bundle;
-import io.dropwizard.Configuration;
 import io.dropwizard.auth.AuthDynamicFeature;
 import io.dropwizard.auth.AuthFilter;
+import io.dropwizard.core.Application;
+import io.dropwizard.core.Configuration;
+import io.dropwizard.core.setup.Bootstrap;
+import io.dropwizard.core.setup.Environment;
 import io.dropwizard.lifecycle.Managed;
 import io.dropwizard.lifecycle.setup.LifecycleEnvironment;
 import io.dropwizard.servlets.tasks.Task;
-import io.dropwizard.setup.Bootstrap;
-import io.dropwizard.setup.Environment;
+import jakarta.ws.rs.Path;
```

Comment:
Is the Jakarta stuff really preferred or is this just because it comes with Dropwizard

Reply:
It's preferred. Javax is basically dead for a long time and there are neither updates to it not it is evolving.

Also Javax doesn't have module-info's so you cannot use it in modularized application.

Reply:
yes, it's very much needed for the open source development.